lib/api/controller/realtime.js

```diff
@@ -54,9 +53,16 @@ class RealtimeController extends NativeController {
     assertHasIndexAndCollection(request);
     assertHasBody(request);
 
+    if (request.input.args.cluster === undefined) {
```

Comment:
Unclear, `replicate` could be better

Reply:
`replicate` or `clusterReplicate` may be more explicit.